README.md

```diff
@@ -9,27 +9,62 @@ Note: Since ray-tracing is very demanding, you better of writing an image to dis
 
 ## Screenshot
 
-![screenshot of a cube rendered with the sample program](/Example/MinGL_Example.PNG?raw=true)
+![screenshot of a cube rendered with the sample program](/Example/MinGL_drawCube_Example.PNG?raw=true)
 
-The above image was drawn using MinGL within a `DrawLine()` function based on [Bresenham’s Line Algorithm](https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm)  
-Bresenham’s Line Algorithm used in the [example](Example/Main.cpp) code was based on https://github.com/ssloy/tinyrenderer/wiki
+The above image was drawn using MinGL `drawLine()` function based on [Bresenham’s Line Algorithm](https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm)  
+Bresenham’s Line Algorithm used in the [example](Example/Main.cpp)'s drawCube() function is based on https://github.com/ssloy/tinyrenderer/wiki
+
+![screenshot of a target rendered with the ShapeSample program](/Example/MinGL_drawShapes_Example.png?raw=true)
+
+The above image was drawn using MinGL functions `drawLine()`, `drawRectangle()`, and `drawCircle()`, the code can be found in [Example.cpp](Example/Example.cpp) drawShapes() function
+
+`drawLine()` function is based on [Bresenham’s Line Algorithm](https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm) 
+
+`drawRectangle()` function utilises `drawLine()` function mentioned above
+
+`drawCircle()` function is based on [Bresenham's Circle Drawing Algorithm](https://www.geeksforgeeks.org/bresenhams-circle-drawing-algorithm/)
 
 Usage
 =====
 
 MinGL depends on [glad](https://glad.dav1d.de/) and [GLFW](https://www.glfw.org/) to work. Following is an example of how files could be organized:
+
 ```
-Main.cpp
-MinGL.h
-MinGL.cpp
-glad/khrplatform.h
-glad/glad.h
-glad/glad.c
-GLFW/glfw3.h
-GLFW/glfw3native.h
+.
+├── Example
+│   ├── Example.cpp
+│   ├── Example.h
+├── glad
+│   ├── glad.c
+│   ├── glad.h
+│   └── khrplatform.h
+├── GLFW
+│   ├── glfw3.h
+│   └── glfw3native.h
+├── MinGL.h
+├── MinGL.cpp
+└── Main.cpp
```

Comment:
Main.cpp is in the Example folder. Please correct this README to reflect it.

Reply:
Done, please tell me if any other change is required.